config-linux.md

```diff
@@ -267,9 +267,9 @@ The following parameters can be specified to setup the controller:
 
 ```json
     "memory": {
-        "limit": 0,
+        "limit": 536870912,
         "reservation": 0,
```

Comment:
I disagree. API should be user-friendly and readable. Having to keep
referring to docs while programming against an API is not a good user
experience. Simple changes like including units in the field names will
save a lot of time in the long run.

On Mon, Apr 11, 2016 at 4:01 PM, W. Trevor King <notifications@github.com>
wrote:

> In config-linux.md
> <https://github.com/opencontainers/runtime-spec/pull/370#discussion_r59296588>
> :
>
> > @@ -267,9 +267,9 @@ The following parameters can be specified to setup the controller:
> >
> >  ```json
> >      "memory": {
> > -        "limit": 0,
> > +        "limit": 536870912,
> >          "reservation": 0,
>
> On Mon, Apr 11, 2016 at 03:52:52PM -0700, Vish Kannan wrote: > @@ -267,9
> +267,9 @@ The following parameters can be specified to setup the
> controller: > > ```json > "memory": { > - "limit": 0, > + "limit":
> 536870912, > "reservation": 0, If we were to be pedantic, I'd rename all
> the fields to include the units. For example, `reservation` becomes
> `reservationBytes`
> Ugh :p. I think that's what docs are for ;).
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/runtime-spec/pull/370/files/c1612af9b80dd828f8fccea3bfca5300281d37b9#r59296588>
>